coffeehouse/lydia/lydia_ai.py

```diff
@@ -6,16 +6,15 @@
 import requests
 
 
-class LydiaAI:
-
-    def __init__(self, coffeehouse_api):
+class LydiaAI(API):
```

Comment:
A lot of packages has already updated to use this new version, making changes like this will break packages again and will require the developers to update their code once again.

I am trying to avoid this.